packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,269 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`StartTime should render LTR correctly 1`] = `
+.c2 {
+  vertical-align: middle;
+  margin: 0 0.25rem;
+  color: #222222;
+  fill: currentColor;
+  width: 0.725rem;
+  height: 0.725rem;
+}
+
+.c4 {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  color: #6E6E73;
+  display: block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.c0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.c1 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding-right: 0.25rem;
+  padding-left: 0.5rem;
+}
+
+.c1 > svg {
+  color: #5A5A5A;
+  margin: 0;
+}
+
+.c5 {
+  border-top: 0.0625rem solid #AEAEB5;
+  width: 100vw;
+  margin-left: 0.625rem;
+  top: 1rem;
+}
+
+.c3 > time {
+  color: #5A5A5A;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c4 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    font-size: 0.8125rem;
+    line-height: 1rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c3 > time {
+    font-size: 0.75rem;
+    line-height: 1rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c3 > time {
+    font-size: 0.75rem;
+    line-height: 1rem;
+  }
+}
+
+<div
+  class="c0"
+>
+  <span
+    class="c1"
+    dir="ltr"
+  >
+    <svg
+      aria-hidden="true"
+      class="c2"
+      focusable="false"
+      height="13"
+      viewBox="0 0 13 13"
+      width="13"
+    >
+      <path
+        d="M6.5 0A6.5 6.5 0 1013 6.5 6.5 6.5 0 006.5 0zm0 11.5a5 5 0 115-5 5 5 0 01-5 5z"
+      />
+      <path
+        d="M7.34 2.9h-1v3.8L9.4 8.57l.41-.56-2.47-1.89V2.9z"
+      />
+    </svg>
+  </span>
+  <span
+    aria-hidden="true"
+    class="c3"
+  >
+    <time
+      class="c4"
+      datetime="2019-08-27"
+    >
+      14:54
+    </time>
+  </span>
+  <div
```

Comment:
what's this div for??

Reply:
This is for the horizontal line after the timestamp

Reply:
And you can write <div class="" dir="" />?? Is that valid html?

Reply:
I have tried using an `<hr />` element in order to avoid having an empty div with a border, but `<hr />` did not work well with FireFox high contrast mode. I will look into other ways how this can be implemented

Reply:
This is now updated and the line is added to an `::after` pseudoelement instead of an additional `div`.